### ISSUE TITLE
bug: missing release notifications

### ISSUE THREAD

**Issue by venkatamutyala** [octobox/octobox#3376]
Are you experiencing this issue in octobox.io or your own instance?
 - [ X] octobox.io

 
If your issue is related to an Octobox.io subscription purchase please ensure you have installed the [Octobox GitHub App](https://github.com/apps/octobox) for the repositories concerned.

🐞 **Problem**

I noticed that I'm not getting all notifications from my projects. It seems like some of the "release" notifications came through while some got missed. Are there some limits to the hosted octobox.io? If so, do you offer private tenancy? or should I host my own?

🎯 **Goal**

I risk missing critical release notifications from various github repos that I am subscribed too.

💡 **Possible solutions**

Just looking for clarity on what's expected with using octobox and if the issues I am having are explicitly because I am using the octobox.io service rather than self-hosted.

📋  **Steps to solve the problem**

 * Comment below about what you've started working on.
 * Add, commit, push your changes
 * Submit a pull request and add this in comments - `Addresses #<put issue number here>`
 * Ask for a review in comments section of pull request
 * Celebrate your contribution to this project 🎉

**Comment by andrew** [comment]
@venkatamutyala sorry about that, was the release notification from a private repo or an open source one? 

There's no private tenancy option at the moment, hosting your own instance via the docker image is the easiest way to deploy it in your own infrastructure.

**Comment by venkatamutyala** [comment]
@andrew the notification was from a public repo. And thanks for confirming on the private tenancy. Curious to know if/what the limits are with the free-tier service. If it's a known scaling issue i'll get migrated off your free service and go self-hosted.

**Comment by andrew** [comment]
I don't believe it's a scaling issue, the main limits are around permissions for accessing extra information about notifications for private repositories, it's definitely a bug/silent failure you ran into but I'm having a hard time reproducing it. 

Do you login in/check octobox.io regularly? If not it can start slowing down the rate it checks your notifications.

**Comment by venkatamutyala** [comment]
Yes, I don't log in more than once a week. But the "Sync" option for an immediate sync should fix that, right? I've been repeatedly checking for the missed notification since it I noticed it and it has yet to appear. Are there any failures I can check for on the github side? As for why I ask about scaling... I have roughly 157 repos that I am subscribed to. Lot's of them being CNF type of projects which result in many notifications during a single day.

**Comment by venkatamutyala** [comment]
Also, just an fyi I am using this account for all my octobox notifications: `github-notifs-venkatamutyala`

The sole purpose of that github account is to help track all the public repositories I am following through your service.
